libbeat/_meta/config.yml.tmpl

```diff
@@ -85,15 +85,25 @@ output.elasticsearch:
 
   # Client Certificate Key
   #ssl.key: "/etc/pki/client/cert.key"
-
+{{end}}
 #================================ Processors =====================================
-
+{{if ne .BeatName "heartbeat"}}
 # Configure processors to enhance or manipulate events generated by the beat.
 
 processors:
   - add_host_metadata: ~
   - add_cloud_metadata: ~
-
+{{else}}
+processors:
+  - add_observer_metadata: 
+  # Optional, but recommended geo settings for the location heartbeat is running in 
+  #geo: 
+    # Token describing this location
+    #name: us-east-1a
+
+    # Lat, Lon "
+    #location: "37.926868, -78.024902"
+{{end}}
```

Comment:
Would it be possible to import a file here? This way we won't have to add conditions for each beat that needs customization, and the customization could be done from each beat magefile.

Reply:
I'm indifferent on the import since this is an odd case. It's much nicer IMHO!

Reply:
I also find this way a nicer. Is there anyone who has objections to this solution apart from @jsoriano?

Reply:
What about something like this to make it a bit more generic?
```
{{if not .UseObserverProcessor}}
# Configure processors to enhance or manipulate events generated by the beat.

processors:
  - add_host_metadata: ~
  - add_cloud_metadata: ~

{{else}}
processors:
  - add_observer_metadata: 
  # Optional, but recommended geo settings for the location {{ .BeatName }} is running in 
  #geo: 
    # Token describing this location
    #name: us-east-1a

    # Lat, Lon "
    #location: "37.926868, -78.024902"
{{end}}

Reply:
Great. I like this idea.